### PR TITLE
fix ColorTypes.U16 warnings

### PR DESCRIPTION
**Pull request by visr**
And directly require julia 0.5

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-  - C:\projects\julia-binary.exe /S /D=C:\projects\julia
-
-build_script:
-# Need to convert from shallow to complete for Pkg.clone to work
-  - IF EXIST .git\shallow (git fetch --unshallow)
-  - C:\projects\julia\bin\julia -e "versioninfo();
-      Pkg.clone(pwd(), \"LasIO\"); Pkg.build(\"LasIO\")"
-
-test_script:
-  - C:\projects\julia\bin\julia --check-bounds=yes -e "Pkg.test(\"LasIO\")"
```
Comment (c42f) [inline appveyor.yml]: Giving up on windows ? :-)

Personally I'm perfectly happy to never use windows, but I've found appveyor to be very easy to use for CI, and it's made some of my other projects more robust on unix as well.  I highly recommend it, if you have half an hour to set it up!

File: REQUIRE [hunk 2]
```diff
@@ -1,5 +1,5 @@
-julia 0.4
-ColorTypes
-FileIO
-GeometryTypes
-Compat
+julia 0.5
+ColorTypes 0.3
+FixedPointNumbers 0.3
```
Comment (c42f) [inline REQUIRE]: Nice.

**Comment by c42f** [comment]
Looks good to me.

As noted, I actually find appveyor CI to be good value for effort.  If you like I could try setting LasIO up on my appveyor account.

**Comment by visr** [comment]
Enabled Appveyor :)
I removed it since I hadn't enabled it yet and was thinking since it is pure julia it doesn't add much. But of course it can still catch errors like the multiline string issue I hit with displaz.jl.

**Comment by c42f** [comment]
Nice.

It seems like it shouldn't make much difference, but in practice I've found
* The win32 build catches bugs like accidentally using `Int64` instead of `Int`, and other architectural oddities.
* Bugs that should have shown up on travis have gone missing on occasion. This was truly odd and I don't remember the specifics!

**Pull request merged**